docs/Documentation/Resource_Management_System/index.md

```diff
@@ -0,0 +1,55 @@
+# Lex Allocation Management System (Beta)
+*Lex is a system for managing your high-performance computing (HPC) resource assignments, account information, and tracking utilization.*
+
+*Note:  Lex is currently in development with beta functionality available.*
```

Comment:
We can remove this now.  Core functionality is available (although it does still say 'Beta' in the top-left).